### ISSUE TITLE
Support `export interface Foo {}`

### ISSUE THREAD

**Issue by phpnode** [facebook/flow#1131]
Whilst un(der)-documented, `interface` is really useful, but it's unfortunate that I can't export an interface directly in the same way that I can `export type T = string;`, as `export interface` is a syntax error.

For now I can work around this limitation with 
```js
interface Foo {};
export type IFoo = Foo;
```

But it'd be much nicer to be able to do it directly:

```js
export interface Foo {};
```

**Comment by jeffmo** [comment]
Fixed in #1145

**Comment by phpnode** [comment]
awesome, thanks!
